nut/Nsps.py

```diff
@@ -227,13 +227,13 @@ def load(fileName='titledb/files.json', verify=True):
 			for k in json.loads(f.read()):
 				_nsp = Fs.factory(k['path'], k['path'], None)
 
-				if not _load_nsp_filesize(k, _nsp) or not _nsp.path:
+				if not _load_nsp_filesize(k, _nsp) or not hasattr(_nsp, 'path') or not _nsp.path:
```

Comment:
What is the purpose of the `hasattr` check here? Will it include any scenarios not covered by the other two conditions?